gateway/gateway.proto

```diff
@@ -131,6 +131,9 @@ message EvaluateRequest {
     string channel_id = 2;
     // The signed proposal ready for evaluation.
     protos.SignedProposal proposed_transaction = 3;
+    // If targeting the peers of specific organizations (e.g. for private data scenarios),
+    // the list of organizations should be supplied here.
+    repeated string endorsing_organizations = 4;
```

Comment:
Technically the orgs put an endorsement on queries so the term is technically correct, but this is counter-intuitive since queries do not get submitted and endorsement policy is therefore not checked. How about calling it target_organization to avoid such confusion?